### ISSUE TITLE
Inconsistent newlines across platforms

### ISSUE THREAD

**Issue by johnwhitington** [ocaml/dune#3762]
#Expected Behavior

Dune screen output the same on different platforms

## Actual Behavior

Dune screen output on OS X / Windows differs

## Reproduction

OS X:

```
gorge:Desktop john$ dune init exe helloworld
Success: initialized executable component named helloworld
gorge:Desktop john$
```

Windows:

```
John Whitington@DESKTOP-OJQME0H ~/helloworld
$ dune init exe helloworld
Success: initialized executable component named helloworld
     <--- **extra newline here**
John Whitington@DESKTOP-OJQME0H ~/helloworld
```

OS X:

```
gorge:helloworld john$ dune build
Info: Creating file dune-project with this contents:
| (lang dune 2.7)
Done: 8/11 (jobs: 1)gorge:helloworld john$ <---- ***missing newline here***
```

Windows:

```
John Whitington@DESKTOP-OJQME0H ~/helloworld
$ dune build
Info: Creating file dune-project with this contents:
| (lang dune 2.7)
Done: 8/11 (jobs: 1)
John Whitington@DESKTOP-OJQME0H ~/helloworld
```

## Specifications

- Version of `dune` (output of `dune --version`): 2.7.0
- Version of `ocaml` (output of `ocamlc --version`) 4.11.1
- Operating system (distribution and version): OS X Catalina / Windows 10 x64 with OCaml from  https://fdopen.github.io/opam-repository-mingw/

**Comment by ian-barnes** [comment]
Linux behaviour is the same as for OS X:
```
ian@ian-laptop:09:52:45:~/hello$ dune init exe helloworld
Success: initialized executable component named helloworld
ian@ian-laptop:09:52:49:~/hello$ dune build
Info: Creating file dune-project with this contents:
| (lang dune 2.7)
Done: 8/11 (jobs: 1)ian@ian-laptop:09:52:53:~/hello$  <---- ***missing newline here***
ian@ian-laptop:09:52:55:~/hello$ dune --version
2.7.0
ian@ian-laptop:09:53:03:~/hello$ ocamlc --version
4.10.0
```

**Comment by nojb** [comment]
There are two issues here as I understand: 1) on Windows, the Cygwin prompt includes a newline prefix which explains the difference between Windows and Linux/OS X, and 2) a missing newline after the status indicator #3737.

1) is a shell configuration issue, not directly related to `dune`, and
2) should have been fixed in #3767.

Closing this, feel free to re-open if needed. Thanks!
